src/js/actions/MesosActions.js

```diff
@@ -0,0 +1,78 @@
+var semver = require("semver");
+
+var AppDispatcher = require("../AppDispatcher");
+var JSONPUtil = require("../helpers/JSONPUtil");
+var MesosEvents = require("../events/MesosEvents");
+
+var MesosActions = {
+  requestVersionInformation: function (host) {
+    JSONPUtil.request(`${host}/version`).then(
```

Comment:
Shouldn't we add Mesos version checks similarly to what we do for other actions to ensure the right endpoint is hit?